_posts/2020-12-03-How-to-create-a-file.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: Creating A File
+description: 
+summary: 
+tags:
+---
+Today is the week after thanksgiving break and Kajari instructed us to work on starting our website. To create a new file, first you have to go to Finder and manually create a file. Then you have to go to terminal and from there you have to type in code so later in sublime text you can see the files inside the folder you have created. Then, you go into sublime text and you can open that folder and you can see where you can start with the coding. WIth this done, you can go into chrome and open file and then open the index.html. 
+
+Once you open projects follow these commands to create an empty index.html file.
+
+* mkdir Thinkment- used to create a folder, not a file.
+* cd Thinkment - Once you have made the folder, open the folder.
+* touch index.html - adds an empty html file so you can start working on the page 
+* touch README.md - this has other files in the directory, this is used as a form of keeping things in one place.
```

Comment:
This one doesn't make any send, you can get rid of this line. `touch README.MD` simply creates an empty file with the name README.MD